### PR TITLE
chore: Sync NPM package version (NR-108685)

### PR DESCRIPTION
**Pull request by bjfield**
The pre-release version number of the NPM package will no longer diverge from the Browser agent version injected via APM agents or by the copy-paste method. All installation methods will now use the same version number.
---

### Overview

- The `VERSION` file has been removed and code relying on it now uses the version from `package.json`.
- The sync-version step has been removed from workflows and the pre-commit hook.
- Workflows have been updated to get their version directly from the `package.json` file.
- References to "pre-release" have been removed from markdown files.

### Related Issue(s)

[NR-108685](https://issues.newrelic.com/browse/NR-108685)

### Testing

All package outputs should match the same version (currently 1.231.0).

### CODE REVIEW

**Review by metal-messiah — APPROVED** [review]

File: .github/workflows/publish-release.yml [hunk 1]
```diff
@@ -91,17 +89,12 @@ jobs:
...
-      - name: get version
-        id: build_versions
-        run: node tools/scripts/get-build-versions.js
       - name: re-publish internal loader (prod)
         run: |
           node ./tools/scripts/upload-ab-to-s3.js \
             --env prod \
             --current https://js-agent.newrelic.com/nr-loader-spa-current.min.js \
-            --next https://js-agent.newrelic.com/nr-loader-spa-${{steps.build_versions.outputs.CDN_VERSION}}.min.js \
+            --next https://js-agent.newrelic.com/nr-loader-spa-$(cat package.json | jq -r '.version').min.js \
```
Comment (patrickhousley) [inline .github/workflows/publish-release.yml:97]: Have you verified the image used by the job container contains these binaries? I just pulled `ubuntu:latest` and it does not have `jq` installed by default. You may want to add it to the `Setup Container` step in all the jobs where it is needed.

Comment (bjfield) [reply .github/workflows/publish-release.yml:97]: Good catch. Thanks!

File: .github/workflows/publish-release.yml [hunk 2]
```diff
@@ -84,24 +82,19 @@ jobs:
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
     steps:
       - name: Setup Container
-        run: apt update && apt install -y git
+        run: apt update && apt install -y git && apt install -y jq
```
Comment (patrickhousley) [inline .github/workflows/publish-release.yml:85]:
```suggestion
        run: apt update && apt install -y git jq
```

File: .github/workflows/publish-release.yml [hunk 3]
```diff
@@ -150,28 +143,23 @@ jobs:
...
```
Comment (patrickhousley) [inline .github/workflows/publish-release.yml:146]:
```suggestion
        run: apt update && apt install -y git jq
```

File: .github/workflows/publish-release.yml [hunk 4]
```diff
@@ -185,7 +173,7 @@ jobs:
       GITHUB_TOKEN: ${{ secrets.DOCS_SITE_FORK_TOKEN }}
     steps:
       - name: Setup Container
-        run: apt update && apt install -y git
+        run: apt update && apt install -y git && apt install -y jq
```
Comment (patrickhousley) [inline .github/workflows/publish-release.yml:176]:
```suggestion
        run: apt update && apt install -y git jq
```

File: .github/workflows/raise-release-notes-pr.yml [hunk 5]
```diff
@@ -12,7 +12,7 @@ jobs:
       GITHUB_TOKEN: ${{ secrets.DOCS_SITE_FORK_TOKEN }}
     steps:
       - name: Setup Container
-        run: apt update && apt install -y git
+        run: apt update && apt install -y git && apt install -y jq
```
Comment (patrickhousley) [inline .github/workflows/raise-release-notes-pr.yml:15]:
```suggestion
        run: apt update && apt install -y git jq
```

**Review by patrickhousley — APPROVED** [review]

**Pull request merged**